content/en/docs/kubernetes/collector/components.md

```diff
@@ -1,11 +1,10 @@
 ---
 title: Important Components for Kubernetes
 linkTitle: Components
-spelling:
-  cSpell:ignore filelog crio containerd logtag gotime iostream varlogpods
-  cSpell:ignore varlibdockercontainers UIDs k8sattributes replicasets
-  cSpell:ignore kubeletstats kubelet cSpell:ignore replicationcontrollers
-  resourcequotas horizontalpodautoscalers cSpell:ignore statefulsets
+spelling: cSpell:ignore filelog crio containerd logtag gotime iostream kubelet
+spelling: cSpell:ignore varlibdockercontainers k8sattributes replicasets 
+spelling: cSpell:ignore kubeletstats resourcequotas horizontalpodautoscalers 
+spelling: cSpell:ignore replicationcontrollers statefulsets varlogpods
```

Comment:
the existing file is broken due to `prettier`, this way it will leave the block as is.

Reply:
Oops, thanks for noticing.
I've submitted a proposed fix. PTAL